Documentation/Signoffs/Communication.md

```diff
@@ -0,0 +1,92 @@
+# Communication Subsystem
+
+**Function:**
+
+![Function](../Images/Communication/Conceptual.PNG)
+
+Figure 1: Communication Subsystem
+
+The goal of this subsystem is to receive and transmit the data from the sensor subsystem to the processor.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The connection between the interceptor and sensors shall be a wireless connection           |Conceptual Design |
+| 2   | The wireless connection shall be password protected to minimize any outside party interference with the intercepting device                |Conceptual Design |
+| 3   | The connection shall operate within the federally defined frequency range of either 2.4 Ghz or 5 Ghz            |Conceptual Design |
+| 4   | The reciever shall be compact enough to use minimal space in the interceptor device which must fit in a one-by-one-by-one box                   |Conceptual Design|
+| 5   | The wireless connection needs to transmit data at a minimum rate of 10.3 Mbps for the golf ball to be defelct                   |Conceptual Design|
```

Comment:
How did you arrive at this number? Give your analysis that lead to this below in the explanation area that you have.